PORT-13415-bug-to-mention-that-user-deprovisioning-in-okta-saml-scim-is-not-visually-displayed-on-ports-ui (#2130)
port-labs/port-docs: docs/sso-rbac/sso-providers/saml/okta.md

<ScimFunctionality/>

> :::info User deprovisioning
> When using SCIM with Okta SAML, if a user is unassigned from the SSO application in Okta, they will automatically lose access to Port. This deprovisioning works at the Auth0 level - meaning the user's ability to access the system is restricted at our login provider. 
> However, this change will not be reflected in the portal's "Users and teams" page.
> :::

Reviewer: ```suggestion
:::info User deprovisioning
When using SCIM with Okta SAML, if a user is unassigned from the SSO application in Okta, they will automatically lose access to Port. Deprovisioning works at the Auth0 level - meaning the user's account will be blocked by the login provider.  
However, this change will not be reflected in the portal's "Users and teams" page.
:::
```

<ScimFunctionality/>

:::info User deprovisioning

> When using SCIM with Okta SAML, user deprovisioning works at the Auth0 level , the user will automatically lose access to Port. 

Reviewer: not sure this is clear. A user will lose access if unassigned to the SSO application, right?
